pipeline/src/indices/venues.ts

```diff
@@ -0,0 +1,34 @@
+export const index = "venues";
+
+export const mappings = {
+  dynamic: "strict",
+  properties: {
+    id: {
+      type: "keyword",
+    },
+    display: {
+      type: "object",
+      enabled: false,
+    },
+    filter: {
+      properties: {
+        id: {
+          type: "keyword",
+        },
+        title: {
+          type: "keyword",
+        },
+      },
+    },
+    nextOpeningDates: {
```

Comment:
Does this mapping need updating to reflect the change to calculate this value when serving from the api rather than at the transform? I think this needs to reflect the types of  `regularOpeningDays` and `exceptionalClosedDays`.

Reply:
didn't use keyword type as we won't use the `display` bit for searching document, only the `filter` part